code/modules/mob/living/carbon/superior_animal/robots/subtype/mining/mining_drone.dm

```diff
@@ -20,6 +20,61 @@
 	var/pickup = TRUE // Do we pick up the ore?
 	var/mining = TRUE // Do we mine rocks?
 
+//Putting this here do to no idea were it would fit other then here
+/mob/living/carbon/superior_animal/robot/mining/verb/toggle_viewRange()
+	set name = "Toggle Viewrange"
+	set desc = "Toggles viewrange on the mining bot, smaller ones makes it more easy to mannully controle the bot, well longer range will be more automatic."
+	set category = "Mob verbs"
+
+	if (viewRange == 6)
```

Comment:
You could use a Switch statement.
Also the current way this is coded, it will directly goes from 6 to 1.

Reply:
Thats what I want